OpenDreamRuntime.Tests/UnitTest1.cs

```diff
@@ -444,5 +445,30 @@ public void UnicodeProcsTest()
 
             Assert.Zero(runtime.ExceptionCount);
         }
+
+        [TestCase("Hello, World!", ", ", -1, 6)]
+        [TestCase("Hello, World!", ", ", 3, 3)]
+        [TestCase("Hello, World!", ", ", 7, 11)]
```

Comment:
Adjust for the previous fixes and add an out-of-bounds test case
```suggestion
        [TestCase("Hello, World!", ", ", -1, 1)]
        [TestCase("Hello, World!", ", ", 3, 3)]
        [TestCase("Hello, World!", ", ", 7, 0)]
        [TestCase("Hello, World!", ", ", 14, 0)]
        [TestCase("Hello, World!", ", ", 0, 0)]
```